### ISSUE TITLE
docker-env script for tcsh is not correct

### ISSUE THREAD

**Issue by ColleIsarco** [kubernetes/minikube#12308]
**Steps to reproduce the issue:** 

1. Login with default shell set to tcsh
2. start minikube on docker: `minikube start --driver=docker`
3. execute command to print out environment: `minikube docker-env`
3. or name shell explicity: `minikube docker-env --shell=tcsh`

**Full output of `minikube logs` command:**
<details>
Not needed here??
</details>

**Full output of failed command:**
<details>
export DOCKER_TLS_VERIFY="1"
export DOCKER_HOST="tcp://192.168.99.100:32771"
export DOCKER_CERT_PATH="/home/andi/.minikube/certs"
export MINIKUBE_ACTIVE_DOCKERD="minikube"

# To point your shell to minikube's docker-daemon, run:
# eval $(minikube -p minikube docker-env)
</details>

But the expected output should look like this:
<details>
setenv DOCKER_TLS_VERIFY "1"
setenv DOCKER_HOST "tcp://192.168.99.100:32771"
setenv DOCKER_CERT_PATH "/home/andi/.minikube/certs"
setenv MINIKUBE_ACTIVE_DOCKERD "minikube"

# To point your shell to minikube's docker-daemon, run:
# eval `minikube -p minikube docker-env`
</details>

In no Go-expert, but it seems that the source-codefor handling a (t)csh-shell differently than bash or similar shells is missing. The docker here is running on a vm, but at least by explicitly naming the shell I expect the correct script.

**Comment by afbjorklund** [comment]
I think support for csh was missed, when the command and `shell` was forked out from docker-machine.

c847979cabe71e6a799f9e15655fc3ff45a63fe0

So `docker-machine env --shell=tcsh` works, but `minikube docker-env --shell=tcsh` does not

Guess nobody uses csh...

**Comment by afbjorklund** [comment]
```console
$ docker-machine env --shell=bash
export DOCKER_TLS_VERIFY="1"
export DOCKER_HOST="tcp://192.168.99.254:2376"
export DOCKER_CERT_PATH="/home/anders/.docker/machine/machines/default"
export DOCKER_MACHINE_NAME="default"
# Run this command to configure your shell: 
# eval $(docker-machine env --shell=bash)
$ docker-machine env --shell=tcsh
setenv DOCKER_TLS_VERIFY "1";
setenv DOCKER_HOST "tcp://192.168.99.254:2376";
setenv DOCKER_CERT_PATH "/home/anders/.docker/machine/machines/default";
setenv DOCKER_MACHINE_NAME "default";
: Run this command to configure your shell: 
: eval `docker-machine env --shell=tcsh`
```

Needs porting over to pkg/minikube/shell

**Comment by sharifelgamal** [comment]
yeah, this is just missing functionality in minikube, should be a pretty simple fix.

**Comment by sharifelgamal** [comment]
should be as simple as adding an entry for tsch in the map at https://github.com/kubernetes/minikube/blob/master/pkg/minikube/shell/shell.go#L60
